recipes/buildbot/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "buildbot" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "cf35d2a87b11aba29c59e47e843deba1be9649114e792e830252ed8f7c35f963"
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  entry_points:
+    - buildbot=buildbot.scripts.runner:run
+    - buildbot_windows_service=buildbot.scripts.windows_service:HandleCommandLine  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools >=8.0
+  run:
+    - autobahn >=0.16.0
+    - jinja2 >=2.1
+    - pyjwt
+    - python
```

Comment:
```suggestion
    - python >3.5
```

Reply:
@marcelotrevisani will it work with the `win` selector?

Reply:
It will not, I didn't see that selector, sorry.
It can't be noarch then